components/single-form.jsx

```diff
@@ -71,7 +71,7 @@ module.exports = React.createClass({
       return (
         <div className="overlay">
           <div className="popup">
-            <h2>
+            <h3>
```

Comment:
The new flow had a bunch of headers changing from h2 to h3. This is hard to do in an ab test, so I styled h2's to be the same values as h3. Now  that the test is committed, all h2s that needed to be styled can now just be made into h3s.

This is pattern 2!